app/controllers/users/emails_controller.rb

```diff
@@ -93,6 +93,7 @@ def permitted_params
 
     def check_max_emails_per_account
       return if EmailPolicy.new(current_user).can_add_email?
+      flash[:email_error] = t('users.emails.error_message')
```

Comment:
We should add some tests to ensure that the flash error message happens. 
You can use this as an example. https://github.com/18F/identity-idp/blob/8246c19ad0cac4fa088d5ddbd092efa251f70f7e/spec/features/phone/add_phone_spec.rb#L106
https://github.com/18F/identity-idp/blob/8246c19ad0cac4fa088d5ddbd092efa251f70f7e/spec/controllers/users/phones_controller_spec.rb#L23